app/containers/Vaults/migrationWhitelist.json

```diff
@@ -158,5 +158,21 @@
     "vaultTo": "0x1C6a9783F812b3Af3aBbf7de64c3cD7CC7D1af44",
     "apyTooltip": "Please migrate funds to our new vault to continue earning boosted yield.",
     "migrationMessage": "This vault is no longer boosted, but is still earning yield. Withdraw funds or migrate your balance to the new v2 Curve UST yVault to continue earning boosted yield."
+  },
+  {
+    "symbol": "crvSUSD",
+    "token": "0xC25a3A3b969415c80451098fa907EC722572917F",
+    "vaultFrom": "0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44",
+    "vaultTo": "0x5a770DbD3Ee6bAF2802D29a901Ef11501C44797A",
+    "apyTooltip": "Please migrate funds to our new vault to continue earning boosted yield.",
+    "migrationMessage": "This vault is no longer boosted, but is still earning yield. Withdraw funds or migrate your balance to the new v2 Curve SUSD yVault to continue earning boosted yield."
```

Comment:
@fameal @dudesahn are these vaults still earning yield? I'm guessing no? @patcito maybe we should just use a generic migrate message (that is not scoped to a vault) in the future.

Reply:
They will be earning for 2 more weeks